tests/test_runner.py

```diff
@@ -0,0 +1,77 @@
+from os import remove
```

Comment:
super nice testing script! It does take some time to set up-- should we reduce the number of shape pars for the test case?

Reply:
The whole test takes less than 1min. I would say it is OK. Also we currently only have `er_band_shift` as a shape parameter. 

The tests spend a large fraction of time on initializing sources. But in real productivity, with the help of `pdf_cache`, this time will be saved.

Reply:
ok